custom_components/huesyncbox/services.py

```
@@ -95,60 +105,51 @@ async def async_set_bridge(call: ServiceCall) -> None:
+                )
+                if ATTR_BRIGHTNESS in target_sync_state
+                else None
+            ),
+            "intensity": target_sync_state.get(ATTR_INTENSITY, None),
+            "hue_target": hue_target,
+        }
+
+        async def set_state(
+            api: aiohuesyncbox.HueSyncBox, **kwargs: Any
+        ) -> None:
+            await api.execution.set_state(**kwargs)  # type: ignore  # noqa: PGH003
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PGH003`)

Remove unused `noqa` directive

(RUF100)